File: crates/provider-import/tests/fixtures/provider.py
from openai import OpenAI

client = OpenAI(
    base_url="https://integrate.api.nvidia.com/v1",
    api_key="nvapi-fixture-secret",
)
client.chat.completions.create(
    model="z-ai/glm-5.2",
    temperature=1,
    top_p=1,
    max_tokens=16384,
    seed=42,
    stream=True,
)

File: crates/provider-import/tests/fixtures/provider.js
import OpenAI from "openai";
const client = new OpenAI({
  baseURL: "http://127.0.0.1:1234/v1",
  apiKey: process.env.LM_STUDIO_API_KEY,
});
await client.chat.completions.create({ model: "local-model", stream: true });
